### PR TITLE
Add nmuennich Sunrise files

### PR DESCRIPTION
**Pull request by nmuennich**
closes #Create a new file for nmuennich Sunrise

### CODE REVIEW

**Comment by James-Mulcahy** [comment]
All Your Base Are Belong To Us

File: nmuennich-Sunrise.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Sunrise, FL
+
+### Recommendations
+- Eat at Thai Spicy
```
Comment (espivack) [inline nmuennich-Sunrise.md]: :taiwan: :fire: !!!

Comment (James-Mulcahy) [reply nmuennich-Sunrise.md]: :hankey:

File: nmuennich-Sunrise.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Sunrise, FL
 
 ### Recommendations
-- Eat at Thai Spicy
+- Eat at Thai Spicy, especially on weekends!
```
Comment (James-Mulcahy) [inline nmuennich-Sunrise.md:5]: Don't have commit(ment) issues!  My Thai II in Plantation!

**Review by James-Mulcahy — COMMENTED** [review]
My Thai II in Plantation.  :hankey:

**Pull request merged**